src/Styleguide/Pages/Artist/NavigationTabs.tsx

```diff
@@ -8,8 +8,9 @@ interface Props {
 }
 
 export const NavigationTabs: React.SFC<Props> = props => {
-  const route = (path = "") => `/${props.artist.id}${path}`
   const { statuses } = props.artist
+  const route = (path = "") => `/artist2/${props.artist.id}${path}`
```

Comment:
@mzikherman - I'm not sure at what point this page will fully launch, but note that we have this namespaced under `artist2` over here and just below